JDBC driver org.apache.derby.jdbc.ClientDriver could not be loaded

**Describe the bug**

ant -f initdb.xml fails with below error for jpa tck (also for security TCK)

BUILD FAILED
/home/jenkins/agent/workspace/jakartaee-tck_master/persistence-tck/bin/initdb.xml:46: The following error occurred while executing this line:
/home/jenkins/agent/workspace/jakartaee-tck_master/persistence-tck/bin/initdb.xml:78: The following error occurred while executing this line:
/home/jenkins/agent/workspace/jakartaee-tck_master/persistence-tck/bin/initdb.xml:51: The following error occurred while executing this line:
/home/jenkins/agent/workspace/jakartaee-tck_master/persistence-tck/bin/initdb.xml:65: Class Not Found: JDBC driver org.apache.derby.jdbc.ClientDriver could not be loaded

Complete test run logs : https://ci.eclipse.org/jakartaee-tck/blue/rest/organizations/jenkins/pipelines/jakartaee-tck/branches/master/runs/1243/nodes/72/steps/271/log/?start=0 

**To Reproduce**
Download https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee9-eftl/staged-910/jakarta-persistence-tck-3.0.0.zip 
Run jpa tck using docker/jpatck.sh

**Additional context**

Derby Information --------
[/home/jenkins/agent/workspace/jakartaee-tck_master/glassfish6/javadb/lib/derby.jar] 10.15.2.0 - (1873585)
[/home/jenkins/agent/workspace/jakartaee-tck_master/glassfish6/javadb/lib/derbytools.jar] 10.15.2.0 - (1873585)
[/home/jenkins/agent/workspace/jakartaee-tck_master/glassfish6/javadb/lib/derbynet.jar] 10.15.2.0 - (1873585)
[/home/jenkins/agent/workspace/jakartaee-tck_master/glassfish6/javadb/lib/derbyclient.jar] 10.15.2.0 - (1873585)
[/home/jenkins/agent/workspace/jakartaee-tck_master/glassfish6/javadb/lib/derbyshared.jar] 10.15.2.0 - (1873585)

From last nights Full Platform TCK TCK run ./root/run_cts.log in https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck/job/master/1275/artifact/ejb30_lite_tx-results.tar.gz:
```
ping.jdbc.pool.check:
     [echo] ts.home = /root/jakartaeetck/bin/xml/../..
     [echo] In a bundle
     [echo] The deliverable currently in use is /root/jakartaeetck/bin/xml/../../bin.

clean.derby.embedded:
     [echo] Deleting embedded derby DB located at /tmp/DerbyDB

derby.embedded.init:
     [echo] init.derby.embedded is true

init.derbyEmbedded:
     [echo] ts.home = /root/jakartaeetck/bin/xml/../..
     [echo] In a bundle
     [echo] The deliverable currently in use is /root/jakartaeetck/bin/xml/../../bin.

check.configure.cmp:

configure.basic:
     [echo] DB URL: jdbc:derby:/tmp/DerbyDB;create=true

BUILD FAILED
/root/jakartaeetck/bin/build.xml:441: The following error occurred while executing this line:
/root/jakartaeetck/bin/xml/impl/glassfish/javaee_vi.xml:107: The following error occurred while executing this line:
/root/jakartaeetck/bin/xml/initdb.xml:137: The following error occurred while executing this line:
/root/jakartaeetck/bin/xml/initdb.xml:505: Class Not Found: JDBC driver org.apache.derby.jdbc.EmbeddedDriver could not be loaded
```

--- a/nucleus/admin/template/src/main/resources/config/server.policy
+++ b/nucleus/admin/template/src/main/resources/config/server.policy
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2013, 2018 Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2013, 2021 Oracle and/or its affiliates. All rights reserved.
  *
  * This program and the accompanying materials are made available under the
  * terms of the Eclipse Public License v. 2.0, which is available at
@@ -128,6 +128,9 @@
       permission java.net.SocketPermission "*", "listen";
       permission java.net.SocketPermission "*", "accept";
       permission java.io.FilePermission       "<<ALL FILES>>", "write,read";
+
+      // Added for Derby 10.15.1.3 and above
+      permission org.apache.derby.shared.common.security.SystemPermission "engine", "usederbyinternals";
 
  };
 //Added for creating stub on JDK 11

